packages/terra-date-picker/src/_variables.scss

```diff
@@ -0,0 +1,4 @@
+// Variables
```

Comment:
Instead of adding a file to handle` $terra-bidi: true` should we update `DatePicker.scss` to use the `terra-margin-start` mixin [here](https://github.com/cerner/terra-core/blob/eccc6f0c81dd2e93419917e5f45fad861c430291/packages/terra-date-picker/src/DatePicker.scss#L20) and [here](https://github.com/cerner/terra-core/blob/eccc6f0c81dd2e93419917e5f45fad861c430291/packages/terra-date-picker/src/DatePicker.scss#L35)